Doc/library/multiprocessing.rst

```diff
@@ -460,9 +460,9 @@ process which created it.
       Traceback (most recent call last):
       Traceback (most recent call last):
       Traceback (most recent call last):
-      AttributeError: 'module' object has no attribute 'f'
-      AttributeError: 'module' object has no attribute 'f'
-      AttributeError: 'module' object has no attribute 'f'
```

Comment:
I think the old message is more user-friendly here.

Reply:
Actually, the old message was incorrect. The correct one is `Can't get attribute 'f' on <module '__main__' (<class '_frozen_importlib.BuiltinImporter'>)>`. I'll fix this in a separate issue (#100408).

Reply:
FTR, this was fixed in #100409